Contrib/CCCS/CalculusOne/03.6/CCD_CCCS_Openstax_Calc1_C1-2016-002_3_6_224.pg

```diff
@@ -49,20 +50,35 @@ $ans3=Formula("$a sec^2(sec($a x)) sec($a x) tan($a x)");
 ###########################
 #  Main text
 
-BEGIN_PGML
+#BEGIN_PGML
 
-Decompose [`\displaystyle y=[$f]`] into two functions [`y=f(u)`] and [`u=g(x)`] such that [`y=f(g(x))`].  Then find `\frac{dy}{dx}`.
+#Decompose [`\displaystyle y=[$f]`] into two functions [`y=f(u)`] and [`u=g(x)`] such that [`y=f(g(x))`].  Then find `\frac{dy}{dx}`.
 
-[`f(u)=`][_______________]{$ans1} [@ AnswerFormatHelp("formulas") @]*
+#[`f(u)=`][_______________] [@ AnswerFormatHelp("formulas") @]*
 
-[`g(x)=`][_______________]{$ans2} [@ AnswerFormatHelp("formulas") @]*
+#[`g(x)=`][_______________] [@ AnswerFormatHelp("formulas") @]*
 
-`\frac{dy}{dx}=`[_______________]{$ans3} [@ AnswerFormatHelp("formulas") @]*
+#`\frac{dy}{dx}=`[_______________] [@ AnswerFormatHelp("formulas") @]*
 
+#END_PGML
```

Comment:
PGML can be retained here, if you wish. One way to accomplish that is similar to what you've done here already: 
- drop the `{$ans}` after the answer blanks for the two composition parts
- split the text after the decomposition parts
- use COMPOSITION_ANS after ending the PGML block
- start a new PGML block after that, and continue with the rest of the problem as usual...